source/common/res/features/resize-inspector/main.js

```diff
@@ -29,8 +29,8 @@
                   ynabToolKit.shared.setToolkitStorageKey('budget-resize-inspector', asideWidth);
                   ynabToolKit.resizeInspector.asideWidth = asideWidth;
 
-                  let wdth = $('.budget-content').css('width');
-                  $('.budget-table-container').css({ width: wdth, top: '9.5rem' });
+                  let bcWidth = $('.budget-content').css('width');
+                  $('.budget-table-container').css({ width: bcWidth, top: '9.5rem' });
```

Comment:
I know this wasn't the best to begin with, but `bcWidth` could be a const instead of a let, and I'd probably just call it `width` in this case so you can use shorthand on the second `.css` call to make it:

```
.css({ width, top: '9.5rem' });
```